tests/csm/rest/test_query_deployment.py

```diff
@@ -42,14 +45,23 @@ def setup_class(cls):
         cls.ha_obj = HAK8s()
         cls.deploy_lc_obj = ProvDeployK8sCortxLib()
         cls.csm_obj = csm_api_factory("rest")
-        cls.csm_conf = configmanager.get_config_wrapper(fpath="config/csm/test_rest_capacity.yaml")
+        cls.csm_conf = configmanager.get_config_wrapper(
+            fpath="config/csm/test_rest_query_deployment.yaml")
         cls.update_seconds = cls.csm_conf["update_seconds"]
+        cls.failed_pod = None
+        cls.kvalue = None
+        cls.deploy_list = cls.csm_obj.master.get_deployment_name(POD_NAME_PREFIX)
```

Comment:
This is no longer required.

Reply:
Check for other places for any redundant code

Reply:
done